queries/ebnf/highlights.scm

```diff
@@ -0,0 +1,47 @@
+;;;; Simple tokens ;;;;
+(terminal) @string.grammar
+
+(special_sequence) @string.special.grammar
+
+(integer) @number
+
+(comment) @comment.block
+
+;;;; Identifiers ;;;;
+(identifier) @variable.grammar
```

Comment:
`@symbol` (or `@symbol.grammar` would also be an option)

Reply:
I agree, changed it to `@symbol.grammar` now